### ISSUE TITLE
Wifi connection drops big files then go to print

### ISSUE THREAD

**Issue by chilito** [SoftFever/OrcaSlicer#6113]
### Is there an existing issue for this problem?

- [X] I have searched the existing issues

### OrcaSlicer Version

2.1.1

### Operating System (OS)

Windows

### OS Version

win 10

### Printer

Flashforge M5

### How to reproduce

1. Send big file (more 50 megs) to print
2. Looking to UPLOAD messege 
3. Got ERROR

### Actual results

Got error and abort print

### Expected results

You need to increase the time to send large files or with a slow connection, at least up to 1 minute, but it’s better to call and choose how long to wait if the Wi-Fi is weak or got large file 

JUST FIX IT
[OrcaSlicer](https://github.com/SoftFever/OrcaSlicer/tree/45224eee217e3590b768d938246dddfdc5f4a12c)/[src](https://github.com/SoftFever/OrcaSlicer/tree/45224eee217e3590b768d938246dddfdc5f4a12c/src)/[slic3r](https://github.com/SoftFever/OrcaSlicer/tree/45224eee217e3590b768d938246dddfdc5f4a12c/src/slic3r)/[Utils](https://github.com/SoftFever/OrcaSlicer/tree/45224eee217e3590b768d938246dddfdc5f4a12c/src/slic3r/Utils)
/TCPConsole.hpp
    void set_defaults()
    {
        m_newline = "\n";
        m_done_string = "ok";
        m_connect_timeout = std::chrono::milliseconds(5000);
        m_write_timeout = std::chrono::milliseconds(10000);
        m_read_timeout = std::chrono::milliseconds(10000);
    }

### Project file & Debug log uploads

any big file

### Checklist of files to include

- [ ] Log file
- [ ] Project file

**Comment by dan4ik123654** [comment]
I also faced this problem

**Comment by psihop** [comment]
Yes. I confirm. The problem exists

**Comment by RusLevel** [comment]
I confirm. There is such a problem.

**Comment by firiat345** [comment]
I also have the same problem.

**Comment by mortonlame** [comment]
Yes. I confirm. The problem exists

**Comment by Bato360** [comment]
Yes, this problem has existed since version 1.9, so it is not possible to use this slicer with the AD5M printer. And the modified weight from FF is generally based on the old version of the slicer 1.8

**Comment by AceAsket** [comment]
Same problem

**Comment by Volkov-Stanislav** [comment]
same problem too

**Comment by hellysu** [comment]
Same problem

**Comment by WebBasedUser** [comment]
Issue still exist in 2.2.0. Any solutions?
